packages/web/src/components/premium-content-purchase-modal/components/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { PayExtraAmountPresetValues } from './types'
+
+export const CUSTOM_AMOUNT = 'customAmount'
+export const AMOUNT_PRESET = 'amountPreset'
+
+export const payExtraAmountPresetValues: PayExtraAmountPresetValues = [
```

Comment:
Nit: consider `as const` instead of the `: PayExtraAmountPresetValues` maybe?

Reply:
Oh yes I keep forgetting about that. I'll give it a try.